esp/platform/espprotocol.cpp

```diff
@@ -112,7 +112,7 @@ const StringBuffer &CEspApplicationPort::getAppFrameHtml(time_t &modified, const
 
     if (needRefresh || embedded_url || !appFrameHtml.length())
     {
-        int passwordDaysRemaining = -1;//-1 means dont display change password screen
+        int passwordDaysRemaining = scPasswordExpired;//-1 means dont display change password screen
```

Comment:
minor: comment is out of date.  It seems slightly counter intuitive that expired means that, but I don't know the context.